protected/humhub/modules/admin/tests/codeception/unit/DatabaseInfoTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace humhub\modules\admin\tests\codeception\unit;
+
+use humhub\modules\admin\controllers\services\DatabaseInfo;
+use tests\codeception\_support\HumHubDbTestCase;
+
+class DatabaseInfoTest extends HumHubDbTestCase
```

Comment:
I guess it is not required to extend from HumHubDbTestCase. `Codeception\Test\Unit`